### PR TITLE
Maybe fix fetchmail when used with pop3

### LINKED ISSUE #2928: fetchmail does not fetch mails in 2.0
### Environment

- [X] docker compose
- [ ] kubernetes
- [ ] docker swarm

### Version

- Version: 2.0, Container-"Id": "sha256:dd0a30f2ecb74491c49c8ab1b1d8d710eb55cc1e81d411acea9171ebd40ff314",

## Description

After updating from 1.9 to 2.0 fetchmails does not fetch mails from remote POP3s-Servers. The container-logs contain the following information with log-level WARNING. Log-level DEBUG does not print more useful information

```
mailu-20-fetchmail-1  | Option --folder is not supported with POP3
mailu-20-fetchmail-1  | Option --folder is not supported with POP3
mailu-20-fetchmail-1  | Option --folder is not supported with POP3
mailu-20-fetchmail-1  | Sleeping for 1000 seconds
mailu-20-fetchmail-1  | fetchmail: Query status=5 (SYNTAX)
mailu-20-fetchmail-1  |
mailu-20-fetchmail-1  |
mailu-20-fetchmail-1  | fetchmail: Query status=5 (SYNTAX)
```
I copied the mailu.db and fired up the new stack based on the newly created configs and docker-compose.yml-files.

## Replication Steps

migrated DB from 1.9 copied to the new folder and starting the stack.

## Observed behaviour

Any chance to get more detailed information from within the container?

## Expected behaviour

Fetching the remote mails and make them available in mailus mail-storage

### PR DESCRIPTION
**Pull request by nextgens**
## What type of PR?

bug-fix

## What does this PR do?

Fix fetchmail when used with pop3.

### Related issue(s)
- closes #2928 

## Prerequisites
Before we can consider review and merge, please make sure the following list is done and checked.
If an entry in not applicable, you can check it or remove it from the list.

- [ ] In case of feature or enhancement: documentation updated accordingly
- [x] Unless it's docs or a minor change: add [changelog](https://mailu.io/master/contributors/workflow.html#changelog) entry file.

### CODE REVIEW

File: optional/fetchmail/fetchmail.py [hunk 1]
```diff
@@ -64,7 +64,7 @@ def run(debug):
                 username=escape_rc_string(fetch["username"]),
                 password=escape_rc_string(fetch["password"]),
                 options=options,
-                folders=folders,
+                folders='' if fetch['protocol'].lower().startswith('pop') else folders,
```
Comment (ghostwheel42) [inline optional/fetchmail/fetchmail.py:67]: You can test for 'pop3', as this is defined very strictly:
`db.Column(db.Enum('imap', 'pop3'), nullable=False)`

**Review by ghostwheel42 — CHANGES_REQUESTED** [review]

File: optional/fetchmail/fetchmail.py (hunk 1 above)
Comment (nextgens) [reply optional/fetchmail/fetchmail.py]: Ok, that's [008ff03](https://github.com/Mailu/Mailu/pull/2929/commits/008ff03882740ffedb260f3daa09e061c34e0ad4)

**Review by ghostwheel42 — APPROVED** [review]
LGTM
This will fix the issue at hand.
We should also "disable" the folders input in the gui though.

**Comment by nextgens** [comment]
bors ping
bors try

**Pull request merged**